### ISSUE TITLE
Suggestion...#Route parameter[s] 

### ISSUE THREAD

**Issue by ncacace** [adonisjs/docs#154]
http://dev.adonisjs.com/docs/4.0/routing
#Route parameters

You have covered a Single Parameter
You have covered a Default Value
Perhaps Multiple Parameters and default values..? The topic heading is after-all Route parameter[s] plural

Best Practice for multiple parameters Restfully maybe...?
https://stackoverflow.com/questions/2602043/rest-api-best-practice-how-to-accept-list-of-parameter-values-as-input

**Comment by ncacace** [comment]
ERRATA    #Routing for SPA's
If you have any other specific routes, **they are be defined before the wildcard route**. For example:
Suggestion:  If you have any other specific routes,
- they should be defined before....
OR
- they are to be defined before....

**Comment by thetutlage** [comment]
Some of your questions have obvious answers but your question seems to be like.

> I know everything but still I’ll ask these questions to make u feel inferior 

If you really want answers, I suggest reframing your questions please

**Comment by ncacace** [comment]
?

Sir,

I refer to your previous email

"Regarding typo’s in documentation, feel free to submit PR here
https://github.com/adonisjs/docs/tree/4.0
<http://PZX9.trk.elasticemail.com/tracking/click?msgid=fl35UPWyTw6x3ibH9Hxx5w2&target=https%3a%2f%2fgithub.com%2fadonisjs%2fdocs%2ftree%2f4.0&v=Tg_ySlsYhFvbMWG3cwP43w2&lc=916668929148174460>

There are many others helping me trying to improve the documentation"

AS I MENTIONED IN A PREVIOUS CORRESPONDENCE

When you have intimate knowledge of a subject matter things appear obvious
to you

Ask yourself this: Is it your intention to write documentation for people
who don't need it and already know how AdonisJS works?

I've been developing since 1996 starting with C++/Java and have experience
in eLearning and have managed projects for Education, Defense and
Commercial sectors from trouble shooting projects bringing them from the
brink of failure to Production Management to doing my own development. (I
may be new to AdonisJS but I've been around the block)

Seriously I was really trying to help and would have preferred to write
directly but you never gave me that option. I apologize if I left you
feeling inferior as this was not my intention at all - On the contrary I
feel it's awesome what you're doing and I have always had respect for the
teams I worked with whether it was the CEO of a multinational company, CTO
to Graphic artists and receptionist.

I assure you that everyone I have the pleasure to work with have always
found me to be very affable and have always found me professional and
respectful.

Maybe you've been doing a couple of late nights and you feeling stressed at
the moment. I think you should read your email again in a couple of days
when you have calmed down and ask yourself if you should expect more from
yourself.

Kind regards,
Nicolas

**Comment by ncacace** [comment]
<img width="723" alt="october7" src="https://user-images.githubusercontent.com/1590147/31409236-7552f91e-ae0b-11e7-894e-9ef41ceb522f.png">

... To clarify - As you will see from your email to me YOU suggested I forward any corrections I found whilst reading your documentation to this link.

They are not questions but some requests for clarification and a correction of an obvious typo.
... Are you now retracting your offer to forward corrections and suggestions?

**Comment by RomainLanz** [comment]
Hey @ncacace 👋  !

> Are you now retracting your offer to forward corrections and suggestions?
Where do you see that?

Your first message in this issue wasn't helping everyone, you were throwing await sentence without any courtesy and without any context.

Reading

> Best Practice for multiple parameters Restfully maybe...?

Or

>You have covered a Single Parameter
>You have covered a Default Value
>Perhaps Multiple Parameters and default values..? The topic heading is after-all Route parameter[s] plural

isn't useful at all!

Please, describe with complete sentence and courtesy what do you want.

**Comment by ncacace** [comment]
<img width="177" alt="screen shot 2017-10-11 at 10 38 46" src="https://user-images.githubusercontent.com/1590147/31430178-636bc75c-ae70-11e7-924c-23c6cbdea24a.png">
Ok - Is this you slapping me....?

Because if it is IT'S CLEAR that your purpose is not to achieve anything constructive at all which renders anything else you say redundant as your purpose is clearly confrontation.
... and more importantly 
a waste of everyone's time - MOST IMPORTANTLY MINE 
(which I gave freely to contribute to this forum)

===================
1. Restful:
It's how the parameters will be introduced. I feel it IS useful 

2. Parameter[s]
I'm sure it would be rare that one single parameter would be introduced to a function so I also feel it IS useful to have an example of several parameters - without it you are just guessing

If the purpose of the documentation is to serve as an addition to Laravel then fine
BUT
If the documentation is to stand on it's own merits no

It would be great for @RomainLanz  to customize the documentation solely for his purposes however it narrows the user base of the framework somewhat.

===================
I would like remind the owner of this forum that if you feel that it's ok for it's members to engage in this sort of unprofessional behavior clearly sets a tone and is hardly the sort of positive environment conducive to successful engagement and development

Again to be REAL CLEAR
Hardminder Virk, I've already stated that I felt that tabling suggestions to documentation made in this type of forum was not very constructive for this VERY reason.
I'm definitely not interested in entering into a Trump-esque trolling competition with Brown-Nosers obvious looking to score a few points with you.
